# Step Slab Part
cvrg_file_paths:
  {
    path_file: !!python/object/apply:string.join [[$(arg pkg_path),"data/csv/step_slab.csv"], /],
    cvrg_path: !!python/object/apply:string.join [[$(arg pkg_path),"data/csv/waypoints.csv"], /], # Path to save the path
    success_flags: !!python/object/apply:string.join [[$(arg pkg_path),"data/csv/success_flags.csv"], /], # Path to save the path
    joint_states: !!python/object/apply:string.join [[$(arg pkg_path),"data/csv/joint_states.csv"], /], # Path to save the trajectory
    mesh_path: !!python/object/apply:string.join [[$(arg pkg_path),"data/meshes/step_slab.stl"], /],
    mesh_viz_path: "package://pct/data/meshes/step_slab.stl",
  }

cvrg_tf_param:
    {
        world_T_part: [1.0,0.0,0.4,0,0,1.57], # x,y,z,rx,ry,rz xyzbxbybz
    }

viz_param:
    {
        rob_speed: 20
    }

gen_path: false
rob_config: [0,0,0,0,0,0]
sampling_res: 5 # Degrees
# The first two values in tolerances are angle
# theta1 and theta2 bounds
# Three bounds in total for X Y and Z
tolerances: [   0,0,0,0,-180,180  ]
max_itr: 200
tol_constraint: 0.2
opt_cost: 6.22
opt_trigger: 0.04
# opt_trigger: 1.0
alpha: 10
neigh_thrld: 0.2
k: 10